yb-voyager/cmd/analyzeSchema.go

```diff
@@ -452,6 +453,16 @@ func checkDDL(sqlInfoArr []sqlInfo, fpath string) {
 			reportCase(fpath, "LANGUAGE C not supported yet.",
 				"", "", "FUNCTION", tbl[2], sqlInfo.formattedStmt)
 			summaryMap["FUNCTION"].invalidCount++
+		} else if regMatch := createTableRegex.FindStringSubmatch(strings.Join(strings.Fields(sqlInfo.stmt), " ")); regMatch !=nil {
```

Comment:
This is a very roundabout way to do a string submatch, your regex handles multiple whitespaces anyways right? Please check if it works without using `Join()` and `Fields()`, and if it does not, please modify the regex to handle this.
P.S. Utilize some publicly available [regex checker](https://regex101.com), only if needed